hazelcast/src/main/java/com/hazelcast/jet/core/ProcessorMetaSupplier.java

```diff
@@ -489,7 +490,7 @@ public void init(@Nonnull Context context) {
                             "Local parallelism of " + context.localParallelism() + " was requested for a vertex that "
                                     + "supports only total parallelism of 1. Local parallelism must be 1.");
                 }
-                String key = StringPartitioningStrategy.getPartitionKey(partitionKey);
+                String key = getPartitionKey(partitionKey);
```

Comment:
static import is less readable, StringPartitioningStrategy is not a default partitioning strategy, so better when it is visible immediately